static/app/views/performance/browser/resources/resourceTable.tsx

```diff
@@ -94,7 +94,12 @@ function ResourceTable({sort}: Props) {
       return <DurationCell milliseconds={row[key]} />;
     }
     if (key === 'span.op') {
-      const opName = row[key] === 'resource.script' ? t('Javascript') : t('Image');
+      const opNameMap = {
+        'resource.script': t('Javascript'),
+        'resource.img': t('Image'),
+        'resource.iframe': t('Javascript (iframe)'),
```

Comment:
l: let's also do `resource.css`, `resource.video`, and `resource.audio`. The rest aren't important enough.

Reply:
done!